Tests/BundleInMemoryTest.php

```diff
@@ -12,16 +12,16 @@ class BundleInMemoryTest extends WebTestCase
 {
     protected static function getKernelClass(): string
     {
-        require_once __DIR__.'/Fixtures/App/AppKernel.php';
+        require_once __DIR__ . '/Fixtures/App/AppKernel.php';
 
         return AppKernel::class;
     }
 
-    protected static function createKernel(array $options = array())
+    protected static function createKernel(array $options = [])
     {
         $class = self::getKernelClass();
 
-        if (!isset($options['test_case'])) {
+        if (! isset($options['test_case'])) {
```

Comment:
This change seems to go against symfony standards. https://symfony.com/doc/current/contributing/code/standards.html